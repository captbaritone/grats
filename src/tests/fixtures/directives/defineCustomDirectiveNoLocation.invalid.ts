/**
 * This is my custom directive.
 * @gqlDirective
 */
function customDirective() {}
