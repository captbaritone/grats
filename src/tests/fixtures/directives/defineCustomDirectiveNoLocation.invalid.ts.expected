-----------------
INPUT
----------------- 
/**
 * This is my custom directive.
 * @gqlDirective
 */
function customDirective() {}

-----------------
OUTPUT
-----------------
src/tests/fixtures/directives/defineCustomDirectiveNoLocation.invalid.ts:3:4 - error: Expected at least one `@on` location for directive definition.

3  * @gqlDirective
     ~~~~~~~~~~~~~
4  */
  ~
